concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+## Motivation
+
+The Enarx project needs to organize its work better, and in a more sustainable way than before. We need to do a better job planning for medium-term work and goals, and in a way that scales more effectively than our prior process.
+
+## Tutorial
+
+Before we start thinking about how our sprints will work, we should lay the groundwork for how our backlog and sprint boards should be organized.
+
+### Triage
+
+By default, we shouldn't do anything when an issue is first submitted. The lack of metadata will indicate it needs to be triaged. (If desired, we could post a quick welcome comment if they're not part of the core team, but that's optional.)
+
+Every standup, we'll look at all issues that are not part of a project and not labeled. This ensures that new issues always get a human response within 24 working hours, and that all issues are organized properly into the backlog -- or, if needed, promoted to the current sprint. It also allows the team to discuss all new issues as they come up.
+
+### Backlog
+
+As our project scales, it’s easy to imagine a single unified backlog becoming incredibly cumbersome and hard to manage. Instead, we should adapt our current approach to area-specific boards, which effectively divides our backlog into categories (SGX, SEV,  infrastructure, etc.).
+
+Each area-specific backlog will be housed in a Github project with the following columns:
+
+- Backlog: for issues we want to tackle eventually, but not a high priority
+- Tracking: for issues that are high-priority, and that we want to take action on sooner rather than later. All new project issues should land here by default to ensure they don’t get lost (automatable with stock Github).
+- Active: for issues that are actively being worked on as part of the current sprint.
+- Done: when things are done, they go here (automatable with stock Github). Mostly to ensure the “Active” column stays organized.
+
+In addition to inherently categorizing our backlog, this approach allows new "categories" to be moved around or created as needed -- and, if we decide to use organization-level projects for our sprints, allows the backlog to be bisected across repositories if wanted. This will support modularization as the project grows.
+
+To start, I propose the following backlog projects:
+
+- Documentation
+- Infrastructure
+- SEV
+- SGX
+- WASM
+
+A few things to consider:
+- Downside: issues that don’t fit squarely into a category don’t have a place to go. Maybe a "Miscellaneous" project?
+- Automation opportunity: we could automatically move issues from Tracking to Active when they are assigned to someone. Automatable with Actions.
+
+### Active Sprint Board
+This board is where the team will track progress on current sprint tasks. **It should hold no backlog;** if an issue or PR is on this board, it is being actively worked on.
+
```

Comment:
Moving issues and PRs between boards is easily done, right? I'm assuming so but I've never tried it myself. Ease of doing this could inform whether we want the component-specific backlog boards.